src/app/profile/addActivityModal/addActivityForm/AddActivityForm.types.ts

```diff
@@ -10,4 +10,18 @@ export type AddActivityFields = {
 
 export type AddActivityFormProps = {
   onSubmit: (data: AddActivityFields) => void;
+  isFromSubmittedSuccessfully: boolean;
+  isSubmitting: boolean;
+  error: string | undefined;
+  onModalClose: VoidFunction;
+};
+
+export type AddActivityFormContainerProps = {
+  name: string;
+  onModalClose: VoidFunction;
```

Comment:
Tu i powyżej - nazwa `onModalClose` na formie jest dość nieszczęśliwa, bo co formowi do modala? On w ogóle nie wie, że modal istnieje, zależności, pamiętasz? Modal wie że ma w sobie forma, ale form jako dziecko nie wie, gdzie może być wykorzystane.
Jak to można inaczej nazwać?

Reply:
onFormSubmission - nie wie co to jest ale ma to wykonać jak `form` jest zatwierdzony z sukcesem.